oidc-ui/src/components/Consent.js

```diff
@@ -255,8 +257,8 @@ export default function Consent({
   };
 
   const handleCancel = (e) => {
+    setCancelPopup(true);
     e.preventDefault();
-    onError("consent_request_rejected", t("consent_request_rejected"));
```

Comment:
You can add this `onError` method calling in `handleDiscontinue` so the user can get an error response when they cancel consent page

Reply:
The changes are done.